internal/view/app.go

```diff
@@ -403,10 +403,7 @@ func (a *App) switchContext(name string) error {
 	a.Halt()
 	defer a.Resume()
 	{
-		ns, err := a.Conn().Config().CurrentNamespaceName()
```

Comment:
I think the current behavior is correct i.e when switching context we want to use the default namespace associated with that context vs using the namespace in the k9s config.  For example if `ctx1` specifies default namespace `ns1` but the k9s config (if it exists!) specifies `ns2` then k9s should use `ns1` as the default namespace. This proposal would always override the namespace with  the k9s config namespace or `default` if not set. 
That said, a case could be made if the new context does not specify a namespace where we could override with the last visited namespace from the k9s config if set....

Does this make sense?

Reply:
Hi, long time no see @derailed .
Is there something I can do to go ahead?